meta-cli/src/cli/list.rs

```diff
@@ -0,0 +1,175 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+use anyhow::{bail, Result};
+use async_trait::async_trait;
+use clap::Parser;
+use common::graphql::Query;
+use common::typegraph::Typegraph;
+use reqwest::Url;
+use std::sync::Arc;
+
+use super::serialize::SerializeReportExt;
+use super::{Action, ConfigArgs};
+use crate::config::{Config, NodeConfig, PathOption};
+use crate::deploy::actors::console::ConsoleActor;
+use crate::deploy::actors::task::serialize::{SerializeAction, SerializeActionGenerator};
+use crate::deploy::actors::task_manager::{StopReason, TaskManagerInit, TaskSource};
+use crate::interlude::*;
+use serde::Deserialize;
+
+#[derive(Parser, Debug)]
+pub struct List {
+    #[clap(long)]
+    max_parallel_loads: Option<usize>,
+}
+
+#[async_trait]
+impl Action for List {
+    #[tracing::instrument]
+    async fn run(&self, args: ConfigArgs) -> Result<()> {
+        let dir = args.dir()?;
+        let config_path = args.config.clone();
+
+        let config = Arc::new(Config::load_or_find(config_path, &dir)?);
+
+        let console = ConsoleActor::new(Arc::clone(&config)).start();
+
+        let action_generator = SerializeActionGenerator::new(
```

Comment:
We will likely have other similar "simple command", do you see how we could simplify that setup into some utils to keep the run DRY?